docs/sections/sidebar.html

```diff
@@ -12,38 +12,54 @@
         <h1>SIDEBAR </h1>
         <div class="row">
           <div class="col-12 col-lg-4">
+
             <aside class="sidebar">
                 <h4 class="mb-3">POPULAR READS</h4>
-                <ul class="list-unstyled">
-                  <li class="mb-3">
-                      <article class="post-card">
-                          <time>04 May, 2020</time>
+                <div id="sidebarCarousel" class="carousel slide">
+                  <ol class="carousel-indicators carousel-indicators--primary mb-0 d-lg-none">
```

Comment:
please note that in Shopify we should implement this dynamically, so based on the number of posts